scripts/add-validator/index.ts

```diff
@@ -41,38 +47,163 @@ class AddValidator {
   public async addValidator() {
     const decimal = +process.env.DECIMAL;
     const bondValue = +process.env.BOND_VALUE * 10 ** decimal;
-    const rewardCommission = +process.env.REWARD_COMMISSION
+    const rewardCommission = +process.env.REWARD_COMMISSION;
     const validator = new Validator();
 
     // Fetch account balance
-    const { stashBalance, controllerBalance } = await validator.accountBalance(this.api, this.stashAccount.address, this.controllerAccount.address);
-    console.log(`The stash account ${this.stashAccount.address} balance is ${stashBalance}`);
-    console.log(`The controller account ${this.controllerAccount.address} balance is ${controllerBalance}`);
+    const { stashBalance, controllerBalance } = await validator.accountBalance(
+      this.api,
+      this.stashAccount.address,
+      this.controllerAccount.address
+    );
+    console.log(
+      `The stash account ${this.stashAccount.address} balance is ${stashBalance}`
+    );
+    console.log(
+      `The controller account ${this.controllerAccount.address} balance is ${controllerBalance}`
+    );
 
     // Generate session key
     const sessionKey = await validator.generateSessionKey(this.api);
     console.log(`The session key is ${sessionKey}`);
 
     // compare stash balance with bond value
     if (stashBalance <= Number(bondValue)) {
-      throw new Error("Bond value needs to be lesser than stash account balance.");
+      throw new Error(
+        "Bond value needs to be lesser than stash account balance."
+      );
     }
 
     // bond token of stash account
-    const bondStashAccount = await validator.bondValue(this.api, this.controllerAccount.address, this.stashAccount, bondValue);
+    const bondStashAccount = await validator.bondValue(
+      this.api,
+      this.controllerAccount.address,
+      this.stashAccount,
+      bondValue
+    );
 
     // set session key
-    const setSessionKey = await validator.setSessionKey(this.api, sessionKey, this.controllerAccount);
+    const setSessionKey = await validator.setSessionKey(
+      this.api,
+      sessionKey,
+      this.controllerAccount
+    );
 
     // set reward commission
-    const reward = await validator.setCommission(this.api, rewardCommission, this.controllerAccount);
+    const reward = await validator.setCommission(
+      this.api,
+      rewardCommission,
+      this.controllerAccount
+    );
+  }
+
+  public async createAccounts() {
+    console.log("Creating Stash and Controller accounts");
+    if (process.env.NETWORK === "mainnet") {
```

Comment:
Lets invert this and check only networks for which we have faucet (devnt, qanet, testnet).

Reply:
Done.